push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -43,12 +47,22 @@
 
     private final Logger logger = LoggerFactory.getLogger(NotificationDispatcher.class);
 
+    /**
+     * Topic to which a "success" message will be sent if a push message was successfully send and "failure" message otherwise.
+     */
+    public static final String KAFKA_PUSH_MESSAGE_SENDING_METRICS_TOPIC = "agpush_pushMessageSendingMetrics";
+    public static final String KAFKA_PUSH_MESSAGE_SENDING_SUCCESS = "agpush_success";
```

Comment:
Me and Dimitra came up with `KAFKA_PUSH_DELIVERY_METRICS_TOPIC`, `KAFKA_METRICS_ON_DELIVERY_SUCCESS` and `KAFKA_METRICS_ON_DELIVERY_FAILURE`. Is that better, @matzew ?